Controllers/Project/MyPlaceController.cs

```diff
@@ -0,0 +1,129 @@
+// using Microsoft.AspNetCore.Authentication.JwtBearer;
+// using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using rexfinder_api.Models;
+using rexfinder_api.Repositories;
+
+namespace rexfinder_api.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+
+public class MyPlaceController : ControllerBase
+{
+    private readonly ILogger<MyPlaceController> _logger;
+    private readonly IMyPlaceRepository _myPlaceRepository;
+
+    public MyPlaceController(ILogger<MyPlaceController> logger, IMyPlaceRepository repository)
+    {
+        _logger = logger;
+        _myPlaceRepository = repository;
+    }
+
+    // [HttpGet]
```

Comment:
Why is this entire method commented out? Could everything but the auth be active?